.github/workflows/copy-workflow.yml

```diff
@@ -62,109 +58,54 @@ jobs:
-        version=$(git describe --tags --abbrev=0 || true) # highest released version on current branch
-        if [[ -n "$version" ]]; then # only deply version.json if there's at least one release
-          printf '{"version": "%s"}' "$version" | jq . > version.json
-          git add version.json
-          git commit -m "add version.json file"
-        fi
-    - name: bump go.mod go version if needed
-      uses: protocol/multiple-go-modules@v1.2
-      with:
-        working-directory: ${{ env.TARGET_REPO_DIR }}
-        run: |
-          # We want our modules to support two Go versions at a time.
-          # As of August 2021, Go 1.17 is the latest stable.
-          # go.mod's Go version declares the language version being used.
-          # As such, it has to be the minimum of all Go versions supported.
-          # Bump this every six months, as new Go versions come out.
-          TARGET_VERSION=1.16
-
-          # Note that the "<" comparison doesn't understand semver,
-          # but it should be good enough for the foreseeable future.
-          CURRENT_VERSION=$(go list -m -f {{.GoVersion}})
-
-          if [[ $CURRENT_VERSION < $TARGET_VERSION ]]; then
-            echo "GO_VERSION_BUMP=1" >> $GITHUB_ENV
-
-            # Update the version in go.mod. This alone ensures there's a diff.
-            go mod edit -go $TARGET_VERSION
-
-            # In the future, "go fix" may make changes to Go code,
-            # such as to adapt to language changes or API deprecations.
-            # This is largely a no-op as of Go 1.17, and that's fine.
-            go fix ./...
-            git add .
-
-            # We don't tidy, because the next step does that.
-            # Separate commits also help with reviews.
-            git commit -m "bump go.mod to Go $TARGET_VERSION and run go fix"
-          fi
-    - name: go mod tidy (on initial deployment and on new Go version)
-      if: ${{ env.INITIAL_TEST_DEPLOYMENT == 1 || env.GO_VERSION_BUMP == 1}}
-      uses: protocol/multiple-go-modules@v1.2
-      with:
-        working-directory: ${{ env.TARGET_REPO_DIR }}
-        run: |
-          go mod tidy
-          if ! git diff --quiet; then
-            git add .
-            git commit -m "run go mod tidy"
-          fi
-    - name: gofmt -s (on initial deployment and on new Go version)
-      if: ${{ env.INITIAL_TEST_DEPLOYMENT == 1 || env.GO_VERSION_BUMP == 1}}
-      working-directory: ${{ env.TARGET_REPO_DIR }}
-      run: |
-        gofmt -s -w .
-        if ! git diff --quiet; then
-          git add .
-          git commit -m "run gofmt -s"
-        fi
-    - name: determine files to add
-      run: echo "FILES=${{ toJson(github.event.inputs.files) }}" >> $GITHUB_ENV
-    - name: add more versioning workflows files, if necessary
+    - name: Run steps specific to go
+      if: matrix.cfg.deploy_go
+      uses: ./template-repo/.github/actions/copy-workflow-go
```

Comment:
I'm not sure if there's a way to use `env.TEMPLATE_REPO_DIR` here.

Reply:
That would be nicer, but maybe GH Actions doesn’t allow it?
Have you tried `${{ env.TEMPATE_REPO_DIR }}`.

Reply:
I added a comment about it. I've also tested the version using env once again in case I did some stupid mistake earlier but GH Actions still doesn't like it: https://github.com/protocol/.github/actions/runs/1583371137